commands/cock.js

```diff
@@ -11,8 +12,21 @@ module.exports = {
         .setRequired(true)
     ),
   async execute(interaction) {
-    await interaction.reply(
-      `${interaction.options.getUser("user")} GOT A COCK ON EM!!!`
-    );
+    inch = getRandomInt(1,13)
+    const user = interaction.options.getUser("user")
+    if (user.id == 687842330006519880){
```

Comment:
If we end up adding more users, it may be worth having a users object with the ID being the key. Just to alleviate some if chaining.
```
const users = {
  553792724373864448: {
     discordName: 'Becker',
     message: 'does not in fact have a cock on them, sadge :('
  }
}
then:

if (users[user.id]) {
  `${interaction.options.getUser("user")} ${users[user.id].message} `
}